lib/mayaUsd/fileio/translators/translatorUtil.h

```diff
@@ -66,13 +72,15 @@ struct UsdMayaTranslatorUtil
     /// empty string, so a typeless def will be generated on export.
     MAYAUSD_CORE_PUBLIC
     static bool CreateDummyTransformNode(
```

Comment:
When creating a dummy transform for Maya reference prims, the resulting Maya transform node stand-in must be transformable, so added a parameter to control transform locking on dummy transform node creation.